portal/patient/templates/PatientListView.tpl.php

```diff
@@ -10,47 +10,50 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-require_once("../../library/options.inc.php");
-$this->assign('title', xlt("Patient Portal") . " | " . xlt("Patient Data"));
-$this->assign('nav', 'patientdata');
-/*
- *  row keys are js underscore camelcase and follow the underscores varables used in this template
- *  $row['city'] or $row['postalCode'] e.t.c.. The keys do not match table columns ie postalCode here is postal_code in table.
- *  */
-$row = array();
+    //require_once ("./../verify_session.php");
```

Comment:
TODO for me
remove above comment to prevent worry in future if we forget when it was added